libdnf/rpm/repo.cpp

```diff
@@ -751,7 +751,11 @@ static std::vector<Key> rawkey2infos(int fd, Logger & logger) {
 
     // set GPG home dir
     char tmpdir[] = "/tmp/tmpdir.XXXXXX";
-    mkdtemp(tmpdir);
+    char * dir = mkdtemp(tmpdir);
+    if (!dir) {
+        throw std::runtime_error("mkdtemp failed");
```

Comment:
I know you're doing a quick fix here, but this isn't a helpful way to handle the error. Ultimately I would guess we should use `std::tmpfile`, and we'd want our own class for the exception? But if you don't want to change to that, why not add a TODO for that and at least include the `strerror` in the error message?